plugins/http_client_plugin/CMakeLists.txt

```diff
@@ -4,4 +4,4 @@ add_library( http_client_plugin
              ${HEADERS} )
 
 target_link_libraries( http_client_plugin appbase eosio_chain fc )
-target_include_directories( http_client_plugin PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" )
+target_include_directories( http_client_plugin PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" "${CMAKE_CURRENT_SOURCE_DIR}/../../libraries/custom_appbase/include" )
```

Comment:
Instead of using relative paths here (and other similar places in this PR), I'd suggest making an custom_appbase `INTERFACE` library and then add it to `target_link_libraries()` here instead (remove the existing appbase). An example would be how CLI11 is done
https://github.com/AntelopeIO/leap/blob/cfa3c11bd5c65e3b070f2ed5adad5c48c04e8922/libraries/cli11/CMakeLists.txt#L1-L6

Reply:
Thanks for the suggestion! It removes the dependencies on custom_appbase's location by its users. Done.